src/components/merge_results/fields/CompareFiles.tsx

```diff
@@ -37,7 +44,13 @@ export const CompareFiles = ({
       <Typography variant="h3" sx={{ display: { xs: 'block', sm: 'none' } }}>
         {isOnImportPage() ? t('basic_data.central_import.import_candidate') : t('unpublished_result')}
       </Typography>
-      <FileBox file={sourceFile} />
+      <FileBox
+        file={sourceFile}
+        shouldShowFileVersion={isCategoryWithFileVersion(
+          sourceResult.entityDescription?.reference?.publicationInstance?.type
+        )}
+        registrationWithFile={sourceFile ? sourceResult : undefined}
```

Comment:
Denne er litt rar, men kom ikke på noen bedre løsning :(
For å laste ned fil må man vite Publication id som filen er koblet til, og når man kopierer filen over til nytt resultat vil den bare være mulig å laste ned i kontekst av det "gamle" resultatet. Sender derfor inn resultatet her bare når filen skal være mulig å laste ned